origin-dapp/src/utils/user.js

```diff
@@ -0,0 +1,14 @@
+import origin from '../services/origin'
+
+const { web3 } = origin.contractService
+
+/**
+ * Takes an Ethereum address and formats it for reliable comparison or display
+ * e.g. 0x627306090abab3a6e1400e9345bc60c78a8bef57 becomes 0x627306090abaB3A6e1400e9345bC60c78a8BEf57
+ * @param {string} an address in any case
+ * @return {string} a case-specific address (currently checksum)
+ */
```

Comment:
nit: document that this function throws:
`@throws {Error} if the input address is not a valid Ethereum address`

Reply:
@franckc I ran into that when letting `null` get passed in a few times. Do you think it would be better to catch the exception or leave as-is and document the warning?

Reply:
I think documenting makes sense to me rather than eating the exception.
It may help us find bugs where address is null but should not...